app/experimenter/experiments/api/v4/serializers.py

```diff
@@ -71,6 +96,16 @@ def get_startDate(self, obj):
         if obj.start_date:
             return obj.start_date.isoformat()
 
+    def get_proposedEnrollment(self, obj):
+        return NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"]["arguments"][
+            "proposedEnrollment"
+        ]
+
+    def get_proposedDuration(self, obj):
+        return NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"]["arguments"][
+            "proposedDuration"
+        ]
```

Comment:
No no we already did this when we create it:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/api/v3/serializers.py#L104-L105

you don't need to do taht here just serialize what's in the db